src/views/Settings.vue

```diff
@@ -157,6 +270,145 @@ export default defineComponent({
       });
       return timeZoneModal.present();
     },
+    async updateFacility(maximumOrderLimit: number) {
+      let resp;
+
+      try {
+        resp = await UserService.updateFacility({
+          "facilityId": this.currentFacility.facilityId,
+          maximumOrderLimit
+        })
+
+        if(!hasError(resp)) {
+          this.currentFacilityDetails.maximumOrderLimit = maximumOrderLimit
+          showToast(translate('Facility updated successfully'))
+        } else {
+          showToast(translate('Failed to update facility'))
+          console.error(resp.data)
+        }
+      } catch(err) {
+        showToast(translate('Failed to update facility'))
+        console.error(err)
+      }
+    },
+    async turnOnFulfillment() {
+      const alert = await alertController.create({
+        header: this.$t('Turn on fulfillment for ', { facilityName: this.currentFacility.name }),
+        buttons: [{
+          text: translate('Cancel'),
+          role: 'cancel'
+        }, {
+          text: translate('Save')
```

Comment:
We could use handler and access data
https://stackoverflow.com/a/55540742/7130371